src/control/server/server.go

```diff
@@ -91,10 +102,30 @@ func Start(log *logging.LeveledLogger, cfg *Configuration) error {
 		return errors.New("NVMe support only available with single server in this release")
```

Comment:
We should be able to remove this error message and the limitation on a single NVMe io_server.  I ran two io_servers with their own NVMe device in primary mode.  Is there some other reason to limit this?

Reply:
Just waiting on #1581 to land first. Until that lands, we can't get rid of the dependency on SPDK in daos_server.